reactioncommerce/reaction
# Dashboard panel keeps re-opening during checkout

Resolves #1953.

Change made: Use proper check for enable shipping on sidebar display

**How to test:**
1. Do a `reaction reset` before starting the app.
2. First, configure shipping, then add an item to cart and proceed to checkout.
3. On the checkout page, close the sidebar (if open). Then, add shipping address and save.
4. Confirm that the sidebar doesn't pop out (since you already have shipping configured.
5. Go back and disable shipping. Go back to checkout (close your sidebar, if opened). 
6. Delete your earlier shipping address (if there).
7. Re-add shipping address and save. Confirm that the sidebar opens to shipping settings (since you don't have it enabled)

You can do multiple variations of trying to checkout with shipping configured and when it is not.
